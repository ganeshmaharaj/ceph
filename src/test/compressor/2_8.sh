#!/bin/sh

#"8K/2/8"
CEPH_ARGS="--bluestore_max_ops 2 --async_compressor_threads 8 --bluestore_compression_min_blob_size 8192 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"16K/2/8"
CEPH_ARGS="--bluestore_max_ops 2 --async_compressor_threads 8 --bluestore_compression_min_blob_size 16384 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"64K/2/8"
CEPH_ARGS="--bluestore_max_ops 2 --async_compressor_threads 8 --bluestore_compression_min_blob_size 65536 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"256K/2/8"
CEPH_ARGS="--bluestore_max_ops 2 --async_compressor_threads 8 --bluestore_compression_min_blob_size 262144 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"4M/2/8"
CEPH_ARGS="--bluestore_max_ops 2 --async_compressor_threads 8 --bluestore_compression_min_blob_size 4194304 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

