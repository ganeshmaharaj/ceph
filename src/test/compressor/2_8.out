Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/8192 in 2123 msec. cores/threads = 2/8
ctx switches 670, use(%) 98, user(%) 97, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (8210 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 10287mb ratio=0.449576 via snappy/8192 in 25641 msec. cores/threads = 2/8
ctx switches 7495, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (31592 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 9458mb ratio=0.41334 via zlib+isal/8192 in 42156 msec. cores/threads = 2/8
ctx switches 12217, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (48081 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7512mb ratio=0.328286 via zlib/8192 in 160743 msec. cores/threads = 2/8
ctx switches 46040, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (166665 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 8086mb ratio=0.353364 via zlib+qat/8192 in 62090 msec. cores/threads = 2/8
ctx switches 5434940, use(%) 71, user(%) 30, kernel(%) 40
[       OK ] PerfTest.perf_compare_qat (71010 ms)
[----------] 5 tests from PerfTest (325558 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (325558 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/16384 in 2008 msec. cores/threads = 2/8
ctx switches 660, use(%) 98, user(%) 97, kernel(%) 0
[       OK ] PerfTest.perf_compare_dummy (7889 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9731mb ratio=0.425288 via snappy/16384 in 25242 msec. cores/threads = 2/8
ctx switches 7365, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (30960 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8862mb ratio=0.387299 via zlib+isal/16384 in 35976 msec. cores/threads = 2/8
ctx switches 10457, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (41700 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7102mb ratio=0.3104 via zlib/16384 in 134676 msec. cores/threads = 2/8
ctx switches 38863, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (140403 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7774mb ratio=0.339746 via zlib+qat/16384 in 38055 msec. cores/threads = 2/8
ctx switches 2975353, use(%) 64, user(%) 30, kernel(%) 33
[       OK ] PerfTest.perf_compare_qat (45007 ms)
[----------] 5 tests from PerfTest (265959 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (265959 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/65536 in 1784 msec. cores/threads = 2/8
ctx switches 595, use(%) 97, user(%) 96, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (7473 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9044mb ratio=0.395235 via snappy/65536 in 27101 msec. cores/threads = 2/8
ctx switches 8067, use(%) 99, user(%) 91, kernel(%) 8
[       OK ] PerfTest.perf_compare_snappy (32611 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8421mb ratio=0.36801 via zlib+isal/65536 in 32416 msec. cores/threads = 2/8
ctx switches 9488, use(%) 99, user(%) 98, kernel(%) 1
[       OK ] PerfTest.perf_compare_isal (37921 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6675mb ratio=0.29174 via zlib/65536 in 139717 msec. cores/threads = 2/8
ctx switches 40637, use(%) 99, user(%) 93, kernel(%) 6
[       OK ] PerfTest.perf_compare_zlib (145217 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7484mb ratio=0.327073 via zlib+qat/65536 in 19641 msec. cores/threads = 2/8
ctx switches 1106904, use(%) 51, user(%) 30, kernel(%) 20
[       OK ] PerfTest.perf_compare_qat (27005 ms)
[----------] 5 tests from PerfTest (250227 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (250227 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/262144 in 2132 msec. cores/threads = 2/8
ctx switches 748, use(%) 98, user(%) 96, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (7812 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.395201 via snappy/262144 in 30788 msec. cores/threads = 2/8
ctx switches 1116757, use(%) 91, user(%) 79, kernel(%) 12
[       OK ] PerfTest.perf_compare_snappy (36345 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8310mb ratio=0.363192 via zlib+isal/262144 in 30755 msec. cores/threads = 2/8
ctx switches 8976, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (36331 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6564mb ratio=0.286855 via zlib/262144 in 139919 msec. cores/threads = 2/8
ctx switches 40623, use(%) 100, user(%) 96, kernel(%) 3
[       OK ] PerfTest.perf_compare_zlib (145450 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7420mb ratio=0.324286 via zlib+qat/262144 in 23436 msec. cores/threads = 2/8
ctx switches 1034336, use(%) 45, user(%) 35, kernel(%) 10
[       OK ] PerfTest.perf_compare_qat (31006 ms)
[----------] 5 tests from PerfTest (256944 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (256944 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/4194304 in 3955 msec. cores/threads = 2/8
ctx switches 1274, use(%) 99, user(%) 98, kernel(%) 0
[       OK ] PerfTest.perf_compare_dummy (9567 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.39519 via snappy/4194304 in 33688 msec. cores/threads = 2/8
ctx switches 61371, use(%) 85, user(%) 79, kernel(%) 5
[       OK ] PerfTest.perf_compare_snappy (39123 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8276mb ratio=0.361688 via zlib+isal/4194304 in 33009 msec. cores/threads = 2/8
ctx switches 9633, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (38519 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6528mb ratio=0.285316 via zlib/4194304 in 138851 msec. cores/threads = 2/8
ctx switches 39893, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (144309 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7399mb ratio=0.323362 via zlib+qat/4194304 in 26506 msec. cores/threads = 2/8
ctx switches 1027840, use(%) 56, user(%) 46, kernel(%) 4
[       OK ] PerfTest.perf_compare_qat (35005 ms)
[----------] 5 tests from PerfTest (266524 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (266524 ms total)
[  PASSED  ] 5 tests.
